src/main/java/com/ericsson/ei/controller/MissedNotificationControllerImpl.java

```diff
@@ -0,0 +1,55 @@
+/*
+    Copyright 2017 Ericsson AB.
+    For a full list of individual contributors, please see the commit history.
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+package com.ericsson.ei.controller;
+
+import java.util.ArrayList;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.CrossOrigin;
+
+import com.ericsson.ei.controller.model.QueryResponse;
+import com.ericsson.ei.queryservice.ProcessMissedNotification;
+
+/**
+ * @author xjibbal
+ *
+ */
+@Component
+@CrossOrigin
+public class MissedNotificationControllerImpl implements MissedNotificationController {
+
+    static Logger log = (Logger) LoggerFactory.getLogger(MissedNotificationControllerImpl.class);
+
+    @Autowired
+    private ProcessMissedNotification processMissedNotification;
+
+    /**
+     * This method is responsible to take the subscriptionName as input and
+     * return the aggregatedObject matching the subscriptionName field.
+     * 
+     * @param subscriptionName
+     * @return ResponseEntity
+     */
+    public ResponseEntity<QueryResponse> getQueryMissedNotifications(String subscriptionName) {
+        ArrayList response = processMissedNotification.processQueryMissedNotification(subscriptionName);
+        log.info("The response is : " + response.toString());
+        return new ResponseEntity(response.toString(), HttpStatus.OK);
+    }
+
```

Comment:
Is this a HTTP GET command and returning back to EI Frontend?
What are you returning, String or Array/List/Json object([<data>]/{<data>}, or if not data []/{}). Returning String(or empty String) "" might cause XML HTTP Errors in Client Web-browser. Returning List/Array JSON valid format solves the XML HTTP Errors. This applies for all entry points that return data to EI Frontend.